src/main/java/HttpProtocal.java

```diff
@@ -0,0 +1,8 @@
+package com.td.HttpServer;
+
+public interface HttpProtocal {
```

Comment:
Similar to your `getRequest` method, the HTTP Protocol (spelling note: all `o`s, no `a`) is a specific thing—it is the definition of rules specifying this method of communication. An HTTP request is a pretty specific thing—why do you feel a need to define an interface for it to implement?